src/app/authentication/sign-in/sign-in.component.html

```diff
@@ -1,14 +1,15 @@
-<h3 class="header">It's great to see you again</h3>
-
 <form #signInForm="ngForm" (ngSubmit)="signInWithEmailAndPassword(signInForm, user)" class="form-grid">
+  <h2 class="header">It's great<br> to see you again!</h2>
+
   <text-input id="Email" name="E-mail" [field]="user.email"></text-input>
   <text-input id="Password" name="Password" [field]="user.password"></text-input>
-  <button standardBtn type="submit">Sign In!</button>
-</form>
 
-<!--<button (click)="signInWithGoogle()">Sign In with Google</button>-->
-<!--<button routerLink="../signup">Don't have an account?</button>-->
-<!--<button routerLink="/dashboard">Kliknij sobie</button>-->
-<!--<button (click)="signOut()">Log out</button>-->
+  <p class="right">Forgot password?</p>
+
+  <div class="btn-container">
+    <button standardBtn type="submit">Sign In!</button>
+    <p>Don't you have an account? <a [routerLink]="['/auth/signup']">Sign Up!</a></p>
```

Comment:
There is no need to use data binding with static path on routerLink. Try to use binding only when you have something to bind, like url variable. For static string paths, just use routerLink='/auth/signup'.